engine/src/main/java/org/terasology/network/NetworkSystem.java

```diff
@@ -21,6 +21,7 @@
 import org.terasology.entitySystem.entity.internal.EngineEntityManager;
 import org.terasology.entitySystem.metadata.EventLibrary;
 import org.terasology.network.exceptions.HostingFailedException;
+import org.terasology.persistence.serializers.EventSerializer;
```

Comment:
You've got some stray unused imports like this one in a few places. If you can get rid of those it helps condense the diff a bit